src/components/SearchSection/index.js

```diff
@@ -104,6 +104,18 @@ const SearchSection = ({
             )}
           </div>
           <div>
+            {session ? (
+              <Button
+                variant="outlined"
+                className={classes.saveButton}
+                href="/account/searches"
+                component={Link}
+                underline="none"
+              >
+                Load search
+              </Button>
+            ) : null}
+
             {session ? (
```

Comment:
A single `{session ?` check should be enough to add both **Load search** and **Save search** buttons